ui/src/app/kvm/views/KVMList/CPUColumn/CPUPopover/CPUPopover.tsx

```diff
@@ -1,22 +1,25 @@
-import PropTypes from "prop-types";
+import type { ReactNode } from "react";
 
 import Popover from "app/base/components/Popover";
+import type { Pod, PodResource } from "app/store/pod/types";
+import { resourceWithOverCommit } from "app/store/pod/utils";
 
 type Props = {
-  allocated: number;
-  children: JSX.Element | string;
-  physical: number;
-  overcommit: number;
+  children: ReactNode;
+  cores: PodResource;
+  overCommit: Pod["cpu_over_commit_ratio"];
 };
 
-const CPUPopover = ({
-  allocated,
-  children,
-  physical,
-  overcommit,
-}: Props): JSX.Element => {
-  const total = physical * overcommit;
-  const free = total - allocated;
+const CPUPopover = ({ children, cores, overCommit }: Props): JSX.Element => {
+  const overCommitted = resourceWithOverCommit(cores, overCommit);
+  const hostCores =
+    cores.allocated_other + cores.allocated_tracked + cores.free;
```

Comment:
Am I right in thinking this is the same calculation that is done in 'CPUColumn'? If so is there are way we can do these calculations in a util or hook so we can just pass a pod and know we're getting the same results everywhere?